README.md

```diff
@@ -222,12 +220,151 @@ on:
+        });
+        return comments.data.find((comment) =>
+          comment.body.includes(botCommentIdentifier)
+        );
+      }
+
+      async function createOrUpdateComment(issueNumber) {
+        if (!issueNumber) {
+          console.log('No issue number provided. Cannot post or update comment.');
+          return;
+        }
+
+        const existingComment = await findBotComment(issueNumber);
+        if (existingComment) {
+          await github.rest.issues.updateComment({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            comment_id: existingComment.id,
+            body: body,
+          });
+        } else {
+          await github.rest.issues.createComment({
+            issue_number: issueNumber,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: body,
+          });
+        }
+      }
+
+      async function logPublishInfo() {
+        console.log('\n' + '='.repeat(50));
+        console.log('Publish Information');
+        console.log('='.repeat(50));
+        console.log('\nPublished Packages:');
+        console.log(packages);
+        console.log('\nTemplates:');
+        console.log(templates);
+        console.log(`\nCommit URL: ${commitUrl}`);
+        console.log('\n' + '='.repeat(50));
+      }
+
+      if (context.eventName === 'pull_request') {
+        if (context.issue.number) {
+          await createOrUpdateComment(context.issue.number);
+        }
+      } else if (context.eventName === 'push') {
+        const pullRequests = await github.rest.pulls.list({
+          owner: context.repo.owner,
+          repo: context.repo.repo,
+          state: 'open',
+          head: `${context.repo.owner}:${context.ref.replace(
+            'refs/heads/',
+            ''
+          )}`,
+        });
+
+        if (pullRequests.data.length > 0) {
+          await createOrUpdateComment(pullRequests.data[0].number);
+        } else {
+          console.log(
+            'No open pull request found for this push. Logging publish information to console:'
+          );
+          await logPublishInfo();
+        }
+      }
+```
+
+This custom script does the following:
+
+- For pull requests: It creates or updates a comment with the publish information.
+- For pushes with an associated open PR: It adds or updates a comment on that PR.
+- For pushes without an open PR (e.g., direct pushes to main): It logs the publish information to the GitHub Actions console.
+
+This is a sample recipe that users can adapt with `--json` and `--comment=off` to create custom comments tailored to their workflow needs.
+
+Remember to set the necessary permissions in your workflow.
+
+This custom approach gives you full control over how pkg.pr.new communicates its results.
+
 ---
 
 > Releasing approved pull requests is the recommended way of having continuous releases. This ensures users always install approved and safe packages.
```

Comment:
This should be above the new example! Because it talks about the examples we had above!